fix(roles): clarify they're additive (#10211)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx

When it comes to what New Relic features your users can access, there are two main factors: 

* A user's **[user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: For organizations on [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), your users' user type is a billing factor. The user type is what sets the maximum allowed capabilities a user can access. It overrides all role-related access. It's meant to be a fairly long-term setting based on someone's expected New Relic duties.

> * A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups). 

Reviewer: Role or roles? Is it possible for a user to have more than one role?

Author: yeah they can have multiple, hence why we have to clarify they're additive.

## Roles [#roles]

> **Roles** are groupings of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities), which are the granular things New Relic users can do (for example, the ability to manage data retention). We have several [default roles](#standard-roles), and Pro and Enterprise edition organizations can create their own custom roles.

Reviewer: ```suggestion
We offer several [default roles](#standard-roles), but organizations with Pro or Enterprise editions can create their own custom roles.
```

Important points about roles: 

> * Roles are additive: users with multiple roles assigned have the total of all the permissions granted by those roles. For example, if you're in a group that gives you the `All product admin` role in an account, and in another group that gives you a `Read only` role for the same account, you have both roles, and are not restricted by the `Read only` role. 

Reviewer: ```suggestion
* Roles are additive: users with multiple roles assigned have the total of all permissions granted by those roles. For example, if you're in a group that gives you the `All product admin` role in an account, and in another group that gives you a `Read only` role for the same account, you have both roles, and are not restricted by the `Read only` role. 
```
